tests/ClientBuilderTest.php

```diff
@@ -140,7 +140,7 @@ public function testRemoveHttpClientPlugin(): void
 
     public function testGetClient(): void
     {
-        $clientBuilder = new ClientBuilder(['dsn' => 'http://public:secret@example.com/sentry/1']);
+        $clientBuilder = new ClientBuilder(new Options(['dsn' => 'http://public:secret@example.com/sentry/1']));
```

Comment:
To simplify the tests I would use the `create` method so that we avoid creating the `Options` object everywhere